src/pages/VoucherRedemption/StoreBanner.tsx

```diff
@@ -10,7 +10,7 @@ interface ContainerProps {
 }
 
 const StoreBanner = (props: Props) => {
-  const { view, voucher } = useVoucherState();
+  const { view, voucher } = useVoucherState(null);
```

Comment:
Do you need to pass null here?

Reply:
yeah for some reason typescript and custom react hooks are not friendly. Had to add null in there, as adding nothing throws a type error